.github/workflows/publish-helm.yml

```diff
@@ -0,0 +1,28 @@
+name: "Publish: helm chart"
+on:
+  push:
+    tags:
+      - '*'
+    paths:
+      - '.github/workflows/publish-helm.yml'
```

Comment:
add `workflow_dispatch:` so we can manually deploy the chart (for testing purposes)

Reply:
done